packages/darma-vt/package.py

```diff
@@ -102,8 +106,6 @@ class DarmaVt(CMakePackage):
     sanity_check_is_dir = ['include/vt']
     sanity_check_is_file = ['cmake/vtConfig.cmake']
     sanity_check_is_file = ['cmake/vtTargets.cmake']
-    sanity_check_is_file = ['cmake/vtTargets-relwithdebinfo.cmake']
-    sanity_check_is_file = ['lib/libvt-relwithdebinfo.a']
```

Comment:
I removed it because it depends on build type, and in this case, I don't see an easy way of concatenating it with the file name.